File: SampelApp/ios/App/App/Plugins/EchoPlugin.m
//
//  EchoPlugin.m
//  App
//
//  Created by Sebastian Vidrea on 13.05.2023.
//

#import <Capacitor/Capacitor.h>

CAP_PLUGIN(EchoPlugin, "Echo",
	CAP_PLUGIN_METHOD(echo, CAPPluginReturnPromise);
)

File: SampelApp/ios/App/App/Plugins/EchoPlugin.swift
//
//  EchoPlugin.swift
//  App
//
//  Created by Sebastian Vidrea on 13.05.2023.
//

import Capacitor

@objc(EchoPlugin)
public class EchoPlugin: CAPPlugin {
	@objc func echo(_ call: CAPPluginCall) {
		let value = call.getString("value") ?? ""
		call.resolve(["value": value])
	}
}
